resources/assets/js/features/tripsList/layouts/TripsList.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import MapModal from '../components/MapModal';
+import PageHeader from '../../../app/components/PageHeader';
+import ListWithTripsItems from '../components/ListWithTripsItems';
+
+
+class TripsList extends Component {
+
+    render() {
+        return (
+            <div>
+                <PageHeader header={ 'Trips list' } />
+                <ListWithTripsItems />
+                <MapModal/>
```

Comment:
you can move MapModal into List component and define open/close modal there instead of global state usage